### PR TITLE
feat:Solve the problem of label type code duplication

### PR DESCRIPTION
**Pull request by kogel-net**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Label.razor.cs [hunk 1]
```diff
@@ -110,14 +110,21 @@ private async Task SubmitLabelAsync(FormContext context)
...
                 {
-                    await LabelCaller.UpdateAsync(_labelModal.Data.Adapt<UpdateLabelDto>());
+                    await LabelCaller.UpdateAsync(dto);
                     await PopupService.EnqueueSnackbarAsync(T("Edit succeeded"), AlertTypes.Success);
                 }
                 else
                 {
-                    await LabelCaller.AddAsync(_labelModal.Data.Adapt<UpdateLabelDto>());
+                    var labels = await LabelCaller.GetLabelsByTypeCodeAsync(dto.TypeCode);
+                    if (labels.Any())
```
Comment (MayueCif) [inline src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Label.razor.cs:122]: 这部分逻辑应该放在service

**Review by MayueCif — COMMENTED** [review]
@kogel-net

File: src/Services/Masa.Dcc.Service/Domain/Label/Services/LabelDomainService.cs [hunk 2]
```diff
@@ -1,19 +1,24 @@
 ﻿// Copyright (c) MASA Stack All rights reserved.
 // Licensed under the Apache License. See LICENSE.txt in the project root for license information.
 
+using Masa.BuildingBlocks.Globalization.I18n;
```
Comment (codding-y) [inline src/Services/Masa.Dcc.Service/Domain/Label/Services/LabelDomainService.cs:4]: Move to global using

File: src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Server/Program.cs [hunk 3]
```diff
@@ -25,7 +25,13 @@
...
 else
 {
-    builder.Services.AddDccApiGateways(c => c.DccServiceAddress = masaStackConfig.GetDccServiceDomain());
+    builder.Services.AddDccApiGateways(c =>
+#if DEBUG
+    c.DccServiceAddress = "http://localhost:6196"
+#else
+    c.DccServiceAddress = masaStackConfig.GetDccServiceDomain()
+#endif
+    );
```
Comment (codding-y) [inline src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Server/Program.cs:34]: Logical repetition

**Review by codding-y — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 4]
```diff
@@ -350,4 +350,6 @@ MigrationBackup/
 .ionide/
 
 .idea/
-.DS_Store
\ No newline at end of file
+.DS_Store
+/src/Services/Masa.Dcc.Service/Assets/Fonts/SourceHanSansCN-Normal.ttf
```
Comment (codding-y) [inline .gitignore:354]: Don't put font files in the project. Can use CDN.

**Review by codding-y — CHANGES_REQUESTED** [review]

File: src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Server/Program.cs [hunk 5]
```diff
@@ -20,19 +20,8 @@
...
-{
-    builder.Services.AddDccApiGateways(c =>
-#if DEBUG
-    c.DccServiceAddress = "http://localhost:6196"
-#else
-    c.DccServiceAddress = masaStackConfig.GetDccServiceDomain()
-#endif
-    );
 }
+builder.Services.AddDccApiGateways(c => c.DccServiceAddress = masaStackConfig.GetDccServiceDomain());
```
Comment (codding-y) [inline src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Server/Program.cs:24]: builder.Services.AddDccApiGateways(c =>	
#if DEBUG	
    c.DccServiceAddress = "http://localhost:6196"	
#else	
    c.DccServiceAddress = masaStackConfig.GetDccServiceDomain()	
#endif	
    );

**Review by codding-y — CHANGES_REQUESTED** [review]

**Review by MayueCif — APPROVED** [review]

File: src/Services/Masa.Dcc.Service/Masa.Dcc.Service.Admin.csproj [hunk 6]
```diff
@@ -60,6 +60,7 @@
 	</ItemGroup>
 	
 	<ItemGroup>
+	  <Folder Include="Assets\Fonts\" />
```
Comment (codding-y) [inline src/Services/Masa.Dcc.Service/Masa.Dcc.Service.Admin.csproj:63]: Why Fonts are needed？

**Review by codding-y — CHANGES_REQUESTED** [review]

**Review by codding-y — APPROVED** [review]

**Pull request merged**